### ISSUE TITLE
Cannot give Meteoriks certificate

### ISSUE THREAD

**Issue by truck** [vsariola/pakettic#18]
So I've got this Meteoriks certificate for you, for pakettic, because "give truck all the things for Finland" is a thing apparently (:   

And I have NO idea how to get it to you (:   

(And I've asked and no one knew how, ok I didn't ask Lackluster, I probably should have done that)

So when talking with Borb about this today, he said he raised an issue and you promptly responded and the resulting headslap could be heard all around the globe.

Also: Hi.  Welcome to my issue.

**Comment by vsariola** [comment]
Haha ok hmm let's see I don't have irc-client installed and your not in discord and I don't want to put my personal info here; can you email me (firstname.lastname@gmail.com; github profile has the real name) and let's discuss there?

Also: Hi truck!

**Comment by truck** [comment]
Yeah, that horrible Discord service banned 2 of my accounts for no reason:  well, ok, the 2nd one was banned because I was using it to be a Revision organizer in 2020.  That's all I used it for and they banned me.  I am not bothering to find out why, they're just crappy.

So yeah I'll mail you (:  And I totally don't want either of our personal infos in here (:
